docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -81,6 +81,14 @@ database depends on the optional [target] parameter. If you need to get the
 list of files corresponding to default build target, do not specify the [target]
 parameter.
 
+Also please make sure ccache (compiler cache) is disabled in your project. 

```

Comment:
"Make sure to disable ccache (compiler cache) in your project before using intercept-build. If ccache is enabled, intercept-build cannot generate the complete compilation database as some compile commands may be skipped if the target objects are already available in the cache. Use the following command to disable ccache before running the intercept-build command:"

Reply:
Approved with the requested changes.